e2e/src/smoke-tests/smoke-test.ts

```diff
@@ -63,6 +63,14 @@ export const smokeTest = (
         `generate @aws/nx-plugin:py#fast-api --name=py-api --no-interactive`,
         opts,
       );
+      await runCLI(
+        `generate @aws/nx-plugin:py#project --name=@e2e-test/py-project --projectType=application --no-interactive`,
+        opts,
+      );
+      await runCLI(
+        `generate @aws/nx-plugin:py#lambda-function --project=@e2e-test/py-api --functionName=my-function --eventType=Any --no-interactive`,
+        opts,
+      );
```

Comment:
I just noticed the e2e tests were failing with this now that the uv --no-annotate bug is fixed:

```
error: Failed to parse: `pyproject.toml`
  Caused by: TOML parse error at line 9, column 18
  |
9 | [tool.uv.sources."e_2_e_test/py_project"]
  |                  ^^^^^^^^^^^^^^^^^^^^^^^
Not a valid package or extra name: "e_2_e_test/py_project". Names must start and end with a letter or digit and may only contain -, _, ., and alphanumeric characters.

```


```suggestion
      await runCLI(
        `generate @aws/nx-plugin:py#project --name=py-project --projectType=application --no-interactive`,
        opts,
      );
      await runCLI(
        `generate @aws/nx-plugin:py#lambda-function --project=e_2_e_test.py_project--functionName=my-function --eventSource=Any --no-interactive`,
        opts,
      );
```